docker/puppetdb/Dockerfile

```diff
@@ -18,6 +18,10 @@ COPY --from=builder /app/target/puppetdb.jar /
 ARG vcs_ref
 ARG build_date
 ARG version="5.1.5"
+# used by entrypoint to submit metrics to Google Analytics;
+# published images should use "production" for this build_arg
+ARG pupperware_analytics_stream="dev"
```

Comment:
I guess since this defaults to `dev`, no need to update on the Azure side?

Reply:
Right.

The only place it's **not** dev is when Distelli builds it and specifies `make build STREAM=production`